dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.json

```diff
@@ -0,0 +1,476 @@
+   "elements": [
+    {
+     "type": "html",
+     "name": "clickTo44",
+     "visible": false,
+     "titleLocation": "hidden",
+     "html": "<h2 style=\"background-color: #ffff00\"> Variable Test</h2>\n<ul>\n<li><span style=\"background-color: #ffff00;\">environment: {environment}</span></li>\n<li><span style=\"background-color: #ffff00;\">userId: {userId}</span></li>\n<li><span style=\"background-color: #ffff00;\">workshopId: {workshopId}</span></li>\n<li><span style=\"background-color: #ffff00;\">workshopCourse: {workshopCourse}</span></li>\n<li><span style=\"background-color: #ffff00;\">workshopSubject: {workshopSubject}</span></li>\n<li><span style=\"background-color: #ffff00;\">regionalPartnerName: {regionalPartnerName}</span></li>\n<li><span style=\"background-color: #ffff00;\">sessionId: {sessionId}</span></li>\n<li><span style=\"background-color: #ffff00;\">submitRedirect: {submitRedirect}</span></li>\n</ul>"
+    },
+    {
+     "type": "html",
+     "name": "clickTo109",
+     "html": "<p><span style=\"font-size: 18pt;\"><strong>{workshopCourse} Summer workshop Post-Survey</strong></span></p>\n<p><strong>Congratulations! You've completed the Code.org {workshopCourse} Summer workshop!</strong></p>\n<p>This survey asks questions about your experience today and the week as a whole.&nbsp;</p>\n<p><strong>Privacy and Confidentiality</strong></p>\n<ul>\n<li>Your individual responses here are completely <strong>confidential.</strong></li>\n<li>Your identity will&nbsp;<strong>never</strong> be revealed publicly or used to tie any responses back to you personally.</li>\n<li>We may share <strong>de-identified,</strong>aggregate data (representing all responses) publicly.</li>\n<li>Your <strong>de-identified&nbsp;</strong>responses will be used by Code.org, our Regional Partners, and our facilitators for program improvement.</li>\n</ul>\n<p>This survey should take about <strong>10 minutes</strong> to complete.&nbsp;&nbsp;<strong>Your complete honesty and candor are appreciated.</strong></p>"
+    },
+    {
+     "type": "html",
+     "name": "clickTo",
+     "html": "<p><strong><span style=\"font-size: 14pt;\">Your {workshopCourse} Workshop Experience</span></strong></p>"
+    },
+    {
+     "type": "matrix",
+     "name": "overall_success",
+     "title": "How much do you agree or disagree with the following statements about the workshop overall?",
+     "columns": [
+      {
+       "value": "1",
+       "text": "Strongly Disagree"
+      },
+      {
+       "value": "2",
+       "text": "Disagree"
+      },
+      {
+       "value": "3",
+       "text": "Slightly Disagree"
+      },
+      {
+       "value": "4",
+       "text": "Neutral"
+      },
+      {
+       "value": "5",
+       "text": "Slightly Agree"
+      },
+      {
+       "value": "6",
+       "text": "Agree"
+      },
+      {
+       "value": "7",
+       "text": "Strongly Agree"
+      }
+     ],
+     "rows": [
+      {
+       "value": "feel more prepared",
```

Comment:
might be placeholders, but values should be formatted like "feel_more_prepared"

Reply:
Got it.  I need to redo ALL of these values in the value/text pairs by hand.  Right now just autogenerated from jotform->surveys.js conversion code.  Needs human intervention.

Reply:
Will do later.